### ISSUE TITLE
Audio is clipped at the end when the audio offset is set

### ISSUE THREAD

**Issue by SleeklyCoding** [motion-canvas/motion-canvas#782]
**Describe the bug**
When we set the 'audio offset' option in the web player then offset is applied at the beginning but at the end, the audio track is clipped and is silence.

**To Reproduce**
Just set audio track for a project and then set 'audio offset' option in the player to check described behavior above.

```ts
import { makeProject } from '@motion-canvas/core';

import example from './scenes/example?scene';
import audio from './audio/audio.mp3';

export default makeProject({
  scenes: [example],
  audio: audio
});

```

**Expected behavior**
Offset should be applied without clipping the end of the audio track.

**Console errors**
N/A

**Package versions:**
- core: 3.10.0
- two: 3.10.1
- ui: 3.10.0
- vitePlugin: 3.10.0

**Additional context**
Screen from the player. There is a silence starting at frame 4802.

![audio-offset-error](https://github.com/motion-canvas/motion-canvas/assets/129587457/c490f5ca-9daf-487f-b56d-4d8249cc89c6)

BTW: When we decide to render this then:

- VLC player skips offset and audio is full without clipping
- When we import rendered file to Davinci Resolve then audio has offset but is clipped.

This is what I noticed. Maybe it will help somehow.

**Comment by aarthificial** [comment]
I was able to reproduce this problem using an `mp3` file but it doesn't happen when using `wav`.
There are [some other known problems with `mp3`](https://github.com/motion-canvas/examples#audio-quality), like the audio being slightly desynchronized, which is why I strongly recommend using `wav` files.

The issue with the offset not working when exporting a video is tracked in https://github.com/motion-canvas/exporters/issues/2
Sadly, I don't have enough knowledge about ffmpeg to fix it

**Comment by r134x7** [comment]
This bug is worse than I expected. Setting audio offset to 0 didn't fix anything because what I found was that you can render 5 minutes of video but, if you have 10 seconds of audio, you are only going to end up with a video that's ten seconds long after rendering five minutes of video because the audio length controls the video length.

**Comment by aarthificial** [comment]
The problem with the in-editor playback should be fixed after #793
When it comes to rendering, I'm closing this in favor of https://github.com/motion-canvas/exporters/issues/2
